x-pack/plugins/security/public/management/users/users_grid/users_grid_page.tsx

```diff
@@ -223,63 +221,67 @@ export class UsersGridPage extends Component<Props, State> {
     };
 
     return (
-      <div className="secUsersListingPage">
-        <EuiPageContent className="secUsersListingPage__content">
-          <EuiPageContentHeader>
-            <EuiPageContentHeaderSection>
-              <EuiTitle>
-                <h1>
-                  <FormattedMessage
-                    id="xpack.security.management.users.usersTitle"
-                    defaultMessage="Users"
-                  />
-                </h1>
-              </EuiTitle>
-            </EuiPageContentHeaderSection>
-            <EuiPageContentHeaderSection>
-              <EuiButton
-                data-test-subj="createUserButton"
-                {...reactRouterNavigate(this.props.history, `/create`)}
-              >
-                <FormattedMessage
-                  id="xpack.security.management.users.createNewUserButtonLabel"
-                  defaultMessage="Create user"
-                />
-              </EuiButton>
-            </EuiPageContentHeaderSection>
-          </EuiPageContentHeader>
-          <EuiPageContentBody>
-            {showDeleteConfirmation ? (
-              <ConfirmDeleteUsers
-                onCancel={this.onCancelDelete}
-                usersToDelete={selection.map((user) => user.username)}
-                callback={this.handleDelete}
-                userAPIClient={this.props.userAPIClient}
-                notifications={this.props.notifications}
+      <>
+        <EuiPageHeader
+          bottomBorder
+          pageTitle={
+            <FormattedMessage
+              id="xpack.security.management.users.usersTitle"
+              defaultMessage="Users"
+            />
+          }
+          description={
+            <FormattedMessage
+              id="xpack.security.management.users.usersDescription"
+              defaultMessage="Manage your users."
+            />
```

Comment:
You don't always need a description if it's not necessary.

Reply:
Yeah I was torn on this. I liked the consistency between the security applications, but I agree that this really isn't helpful